### PR TITLE
misc. small fixes

### PR DESCRIPTION
**Pull request by jimshur**

### CODE REVIEW

File: src/main/java/org/bigml/binding/localmodel/Tree.java [hunk 1]
```diff
@@ -975,26 +975,31 @@ public Prediction predict(final JSONObject inputData, List<String> path,
...
                 long totalInstances = calculateTotalInstances(distribution);
 
-                return new Prediction(((JSONArray) distribution.get(0)).get(0),
-                        wsConfidence(((JSONArray) distribution.get(0)).get(0), distribution,
-                                totalInstances, DEFAULT_RZ),
-                        totalInstances, null,
-                        path, distribution, "categorical",
-                        lastNode.getTree().getChildren(), null, null);
+                if (distribution.size() > 0) {
+                    return new Prediction(((JSONArray) distribution.get(0)).get(0),
```
Comment (javinp) [inline src/main/java/org/bigml/binding/localmodel/Tree.java]: move line 990 inside this if?

Comment (jimshur) [reply src/main/java/org/bigml/binding/localmodel/Tree.java]: Yes, and 989 too. I'll update the PR.

Comment (javinp) [reply src/main/java/org/bigml/binding/localmodel/Tree.java]: 989 ???

Comment (jimshur) [reply src/main/java/org/bigml/binding/localmodel/Tree.java]: Of course not, what fool would suggest that? ... Me!  Never mind. Just 990!

**Pull request merged**